packages/console/src/pages/Connectors/index.module.scss

```diff
@@ -1,7 +1,9 @@
 @use '@/scss/underscore' as _;
 
-.tabs {
-  margin-top: _.unit(4);
+.container {
+  .tabs {
+    margin-top: _.unit(4);
+  }
 }
 
 .connectorName {
```

Comment:
Wrap all classes in the new `.container`?

Reply:
updated.